frontend/src/lib/Main.svelte

```diff
@@ -2,11 +2,15 @@
   import { Button } from 'carbon-components-svelte'
   import { SkeletonPlaceholder } from 'carbon-components-svelte'
   import { currentWord, randomize, setLanguage } from './stores'
+  import languages from '../languages/languages.json'
+  import NotFound from './NotFound.svelte'
 
   import 'carbon-components-svelte/css/g10.css'
```

Comment:
Remove these if they're already in the `App`

Reply:
Should be resolved.